public/app/plugins/datasource/tempo/datasource.ts

```diff
@@ -353,7 +353,7 @@ export class TempoDatasource extends DataSourceWithBackend<TempoQuery, TempoJson
 
     return {
       ...expandedQuery,
-      query: this.templateSrv.replace(query.query ?? '', scopedVars),
+      query: this.templateSrv.replace(query.query ?? '', scopedVars, 'pipe'),
```

Comment:
I think it would make sense to use VariableFormatID.Pipe enum here instead of the raw string.